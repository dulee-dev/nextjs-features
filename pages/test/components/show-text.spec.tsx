import { render, screen } from '@testing-library/react'
import ShowText from './show-text'

describe("ShowText", () => {
  it("", () => {
    const { container } = render(<ShowText text='hello world!'/>);
    const tree = container
    expect(tree).toMatchSnapshot();
  })
})