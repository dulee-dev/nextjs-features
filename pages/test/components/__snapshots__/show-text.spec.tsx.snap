// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowText  1`] = `
<div>
  <div
    class="text"
  >
    hello world!
  </div>
</div>
`;
